packages/plugin-core/src/commands/base.ts

```diff
@@ -56,6 +57,9 @@ export abstract class BaseCommand<
   async run(args?: Partial<TRunOpts>): Promise<TOut | undefined> {
     // @ts-ignore
     const ctx = `${this.__proto__.constructor.name}:run`;
+
```

Comment:
to make this slightly more useful, can we track two things for commands?

1. duration for invocation. see [here](https://github.com/dendronhq/dendron/blob/feat/anchor-auto-complete/packages/plugin-core/src/_extension.ts#L393:L393) for example
2. error event. basically add a finally clause and set error to true if there was an error